src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -209,21 +209,76 @@ public void clickVisibilityOptionForResponseCommentAndSave(String idString, int
         commentEditForm.findElement(By.className("col-sm-offset-5")).findElement(By.tagName("a")).click();
         ThreadHelper.waitFor(1000);
     }
+    
+    /**
+     * Makes sure the result panels are indeed all visible.
+     */
+    public void verifyResultsVisible() {
+        assertTrue(areResultsVisible());
+    }
+    
+    /**
+     * Makes sure the result panels are indeed all hidden.
+     */
+    public void verifyResultsHidden() {
+        assertTrue(areResultsHidden());
+    }
 
-    public boolean verifyAllResultsPanelBodyVisibility(boolean visible) {
-        int numOfQns = browser.driver.findElements(By.cssSelector(".panel-heading+.panel-collapse")).size();
-        assertTrue(numOfQns > 0);
-
-        // Wait for the total duration according to the number of collapse/expand intervals between questions
-        ThreadHelper.waitFor((numOfQns * 50) + 1000);
+    /**
+     * Checks if the body of all the result panels are visible
+     * @return true if all result panel body are visible
+     */
+    public boolean areResultsVisible() {
+        return isAllResultsPanelBodyVisibilityEquals(true);
+    }
 
-        for (WebElement e : browser.driver.findElements(By.cssSelector(".panel-heading+.panel-collapse"))) {
-            if (e.isDisplayed() != visible) {
+    /**
+     * Checks if the body of all the result panels are hidden
+     * @return true if all result panel body are hidden
+     */
+    public boolean areResultsHidden() {
+        return isAllResultsPanelBodyVisibilityEquals(false);
+    }
```

Comment:
These two `areResults*` functions are called only by their respective `verifyResults*` counterparts here, so consider making them private.
Alternatively, you can move the entire condition to the `verifyResults*`, something like:
```java
public void verifyResultsVisible() {
    assertTrue(isAllResultsPanelBodyVisibilityEquals(true));
}
```